### PR TITLE
Add menu bar to task page

### PR DESCRIPTION
**Pull request by apoorvadhanala**
Add menu bar to task page allowing user to access other pages from to do list page

### CODE REVIEW

**Review by SeanT18 — APPROVED** [review]

**Comment by sidharthp** [comment]
Could you add a description to the PR?

File: src/main/webapp/task.html [hunk 1]
```diff
@@ -7,16 +7,31 @@
...
 
-    <hr/>
+        <hr/>
 
-    <ul id="task-list"></ul>
+        <ul id="task-list"></ul>
 
-  </body>
+    </body>
 </html>
```
Comment (sidharthp) [inline src/main/webapp/task.html]: nit: Add newline at the end of the file.

**Review by sidharthp — APPROVED** [review]

File: src/main/webapp/task.html [hunk 2]
```diff
@@ -7,16 +7,31 @@
...
     <body onload="loadTasks()">
-    <h1 class="task-title">TO-DO LIST</h1>
-    <form method="POST" action="/new-task">
-        <input type="text" name="title" />
-        <br/>
-        <button>Submit</button>
-    </form>
+        <div class="top-container">
+            <div class="dropdown">
+                <button class="dropbtn">MENU</button>
```
Comment (yjennywang) [inline src/main/webapp/task.html:12]: Fyi, you can actually do capitalization via an allcaps property: https://www.w3schools.com/cssref/pr_text_text-transform.asp

**Comment by yjennywang** [comment]
> Could you add a description to the PR?

And also, how you have tested the code? (e.g. ran it through an HTML validator, screenshot...)

**Comment by apoorvadhanala** [comment]
> > Could you add a description to the PR?
> 
> And also, how you have tested the code? (e.g. ran it through an HTML validator, screenshot...)

I'll start including screenshots in the PR going forward!

**Pull request merged**